tooling/template-base/components/Navigation/CurrentUserNavBar.tsx

```diff
@@ -36,7 +36,7 @@ export default function CurrentUserNavBar() {
         </div>
 
         <div className="hidden md:block">
-          <button className="hidden md:block">
+          <div className="hidden md:block">
```

Comment:
See my comment in #149 on this. Can also just omit this change from this PR since it's more appropriately addressed there.